.github/workflows/docker-build-test.yml

```diff
@@ -0,0 +1,19 @@
+# workflow to test whether we can build the Dockerfile
+# from https://stackoverflow.com/questions/67134410/how-to-test-my-dockerfile-for-my-python-project-using-github-actions
+
+name: Docker build and test
+
+on:
+  push:
+# you can trigger on anything you want
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build Docker image
+        run: docker build -t prql .
+      # - name: Run tests inside the container
+      #   run: docker run samplepy poetry run pytest
```

Comment:
Nice, this is good to have.

Let's add it as a job to `test-all.yaml`? That's where these wider tests go; we don't need a separate workflow for each

Reply:
Again, I am happy to have you do this. Thanks.

Reply:
Ok, though then let's remove it from this PR?